gamestonk_terminal/portfolio_optimization/port_opt_api.py

```diff
@@ -18,11 +18,40 @@ def optimize(list_of_stocks:List, other_args:List[str]):
         Dictionary of weights where keys are the tickers
 
     """
+
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        prog="optimize",
+    )
+
+    parse
+
     weights = {}
     n_stocks = len(list_of_stocks)
     for stock in list_of_stocks:
         weights[stock] = round(1/n_stocks, 5)
 
     return weights
 
+def equal_weight(list_of_stocks:List[str]):
+    """
+        Equally weighted portfolio, where weight = 1/# of stocks
+
+        Parameters
+        ----------
+        list_of_stocks: List[str]
+            List of tickers to be included in optimization
+
+        Returns
+        -------
+        weights : dict
+            Dictionary of weights where keys are the tickers
+
+        """
+    weights = {}
+    n_stocks = len(list_of_stocks)
+    for stock in list_of_stocks:
+        weights[stock] = round(1 / n_stocks, 5)
+
+    return weights
```

Comment:
You have 2 duplicated functions here.

I would probably call this file like "mean_variance_optimizer_view" and add the methods that correspond to this category inside.

Reply:
yeah I just added a second one.  This is still a draft so I need to clean up.